Contracts/Contracts.Tests/TesteContracteTerti/TestData.cs

```diff
@@ -62,50 +62,83 @@ public static ContractTerti GenerateContractTerti(string id)
                         SemnaturaResponsabilContract = false,
                         Total = 20,
                         DeplasareList = new List<Deplasare>
-                    {
-                        new Deplasare
-                        {
-                            Id = "642cd4c0-0ed6-4a84-9942-24ed1fab704a",
-                            LoculDeplasarilor = "fdas",
-                            ScopulDeplasarilor = "fsa",
-                            Valoarea = 10
-                        },
-                        new Deplasare
-                        {
-                            Id = "b311add7-ea20-4940-a705-5146c85aa324",
-                            LoculDeplasarilor = "abc",
-                            ScopulDeplasarilor = "cba",
-                            Valoarea = 10
-                        }
-                    }
-                    },
-                    StateDeFunctii = new StateDeFunctii()
-                    {
-                        SemnaturaDirectorDepartament = true,
-                        SemnaturaResponsabilContract = false,
-                        StatDeFunctiiList = new List<StatDeFunctii>
                         {
-                            new StatDeFunctii
+                            new Deplasare
                             {
-                                Id = "05972b6c-5104-4723-8c6f-6f4a3b65e35e",
-                                Nume = "Mircea",
-                                GradDidactic = "Profesor",
-                                DataInceperiiActivitatii = DateTime.Parse("12/05/2018"),
-                                DataFinalizariiActivitatii = DateTime.Parse("12/08/2018"),
+                                Id = "642cd4c0-0ed6-4a84-9942-24ed1fab704a",
+                                LoculDeplasarilor = "fdas",
+                                ScopulDeplasarilor = "fsa",
+                                Valoarea = 10
                             },
-                            new StatDeFunctii
+                            new Deplasare
                             {
-                                Id = "b32769bf-98fe-4de3-ab08-4ad4b0df3dcf",
-                                Nume = "Mihai",
-                                GradDidactic = "Decan",
-                                DataInceperiiActivitatii = DateTime.Parse("11/06/2014"),
-                                DataFinalizariiActivitatii = DateTime.Parse("10/06/2015"),
+                                Id = "b311add7-ea20-4940-a705-5146c85aa324",
+                                LoculDeplasarilor = "abc",
+                                ScopulDeplasarilor = "cba",
+                                Valoarea = 10
                             }
                         }
-                    }
+                    },
+                    StateDeFunctii = new StateDeFunctii(),
```

Comment:
```suggestion
                    StateDeFunctii = new StateDeFunctii(),
```
Din cauza asta nu iti merg testele unitare. Daca te uiti pe master aici sunt mai multe date